dbms/src/Storages/DeltaMerge/FilterParser/FilterParser.cpp

```diff
@@ -130,7 +128,7 @@ inline RSOperatorPtr parseTiCompareExpr( //
     /// Only support `column` `op` `literal` now.
 
     Attr attr;
-    Field value;
+    std::vector<Field> values;
```

Comment:
I dont' understand why here we need to use values? could you give an example here? I think we check the expr.children_size = 2 here, and one will be column expr and the other one is literal expr here.

Reply:
bug orz.... For `in` operator, `expr` is like `[column, literal1, literal2......]`
Fixed.

Reply:
could you please add some these cases in the unit test?

Reply:
done